src/js/views/right_panel_view.js

```diff
@@ -631,11 +631,11 @@
                 min: 100,
                 value: self.zoom_avg,
                 slide: function(event, ui) {
-                    self.update_img_css(ui.value, 0, 0);
-                    $('#vp_zoom_value').val(ui.value);
+                    self.update_img_css(parseInt(ui.value), 0, 0);
+                    $('#vp_zoom_value').val(parseInt(ui.value));
                 },
                 stop: function( event, ui ) {
-                    self.zoom_avg = ui.value;
+                    self.zoom_avg = parseInt(ui.value);
```

Comment:
You don't actually need `parseInt()` for the `slide` and `stop` functions since the `ui.value` will already be a Number.

Reply:
Ok, I remove it from these two functions